install/installer/pkg/components/ws-manager-bridge/deployment.go

```diff
@@ -198,6 +199,16 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 								},
 								volumeMounts...,
 							),
+							LivenessProbe: &corev1.Probe{
+								ProbeHandler: corev1.ProbeHandler{
+									HTTPGet: &corev1.HTTPGetAction{
+										Path: "/healthz",
+										Port: intstr.FromInt(9090),
+									},
+								},
+								InitialDelaySeconds: 15,
+								PeriodSeconds:       20,
```

Comment:
@svenefftinge How often do we retry here? What's the timeout? What the error threshold?

IMO we should fully specify this to make it easier for future-us to operate this w/o looking up the k8s docs every time.

I'd suggest the following values:
```
FailureThreshold: 3,
SuccessThreshold: 1,
TimeoutSeconds:   2,
PeriodSeconds:   20,
InitialDelaySeconds: 15,
```